### PR TITLE
Add FPC Source v2.6.4

### PR DESCRIPTION
**Pull request by Gutem**
Lazarus dependencies

### CODE REVIEW

File: Casks/fpc-source.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'fpc-source' do
+  version '2.6.4-20140421'
+  sha256 'fa7fdfff220241de9e11222e159bcc1b6f3c82588d437191666b5b0ec5fb51cf'
+
+  # sourceforge.net is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/fpc-source.rb]: On a phone now, so can’t yet confirm this.

**Comment by vitorgalvao** [comment]
It’s missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#uninstall-stanza-details) geared for that.

File: Casks/fpc-source.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/fpc-source.rb]: Downloads I see point to `downloads.sourceforge.net/freepascal`, not `lazarus`, please clarify.

Comment (Gutem) [inline Casks/fpc-source.rb]: Really? 
`curl http://downloads.sourceforge.net/lazarus/fpcsrc-2.6.4-20140421-i386-macosx.dmg` shows me a redirect to 
`http://downloads.sourceforge.net/project/lazarus/Lazarus Mac OS X i386/Lazarus 1.4.2/fpcsrc-2.6.4-20140421-i386-macosx.dmg`

Comment (vitorgalvao) [inline Casks/fpc-source.rb]: I mean on http://www.freepascal.org/. `lazarus` seems to be a separate project.

Comment (Gutem) [inline Casks/fpc-source.rb]: No, it's not. Lazarus is the Cross Platform IDE for Free Pascal.
You can check it out at the Wiki: `http://wiki.freepascal.org/` or direct in the Lazarus's OSX Install info page: `http://wiki.lazarus.freepascal.org/Installing_Lazarus_on_MacOS_X#Installation_from_disk_images`, where they specify FPC and FPCSRC as dependences and clicking in download you will redirect to Sourceforge's page of Lazarus, not to the Free Pascal.
Pay attention to `lazarus.freepascal.org` at Wiki's URL.

Comment (vitorgalvao) [inline Casks/fpc-source.rb]: Found it. Why aren’t you calling this `fpcsrc`, though?

**Pull request closed without merging**